packages/core/src/components/forms/_form-group.scss

```diff
@@ -0,0 +1,144 @@
+  Note that this feature necessitates additional DOM elements, thus requiring a more complex structure achieved with `.pt-form-group`.
+</div>
+
+- Link each label to its respective control element with a `for={#id}` attribute on the `<label>` and
+`id={#id}` on the control.
+
+- Add `.pt-intent-*`, `.pt-disabled`  to `.pt-form-group` to style the label and helper text (similar to labels, nested controls need to be styled separately).
+
+- Add `.pt-inline` to `.pt-form-group` to place the label to the left of the control.
+
+- Add `.pt-large` to `.pt-form-group` to align the label when used with large Blueprint controls.
+
+Markup:
+<div class="pt-form-group">
+  <label class="pt-label" for="example-form-group-input-a">
+    Label A
+    <span class="pt-label-description">(required)</span>
+  </label>
+  <div class="pt-form-content">
+    <div class="pt-input-group">
+      <span class="pt-icon pt-icon-calendar"></span>
+      <input id="example-form-group-input-a" class="pt-input" style="width: 200px;" type="text" placeholder="Input group" dir="auto" />
+    </div>
+    <div class="pt-form-helper-text">Helper text with details / user feedback</div>
+  </div>
+</div>
+<div class="pt-form-group pt-intent-danger">
+  <label class="pt-label" for="example-form-group-input-b">
+    Label B
+    <span class="pt-label-description">(required)</span>
+  </label>
+  <div class="pt-form-content">
+    <input id="example-form-group-input-b" class="pt-input pt-intent-danger" style="width: 300px;" placeholder="Input text" type="text" dir="auto" />
+    <div class="pt-form-helper-text">Please enter a value</div>
+  </div>
+</div>
+<div class="pt-form-group pt-inline pt-large pt-disabled">
+  <label class="pt-label" for="example-form-group-input-c">
+    Label C
+    <span class="pt-label-description">(required)</span>
+  </label>
+  <div class="pt-form-content">
+    <div class="pt-input-group pt-large pt-disabled">
+      <span class="pt-icon pt-icon-calendar"></span>
+      <input id="example-form-group-input-c" class="pt-input" disabled style="width: 200px;" type="text" placeholder="Input group" dir="auto" />
+    </div>
+    <div class="pt-form-helper-text">Helper text with details / user feedback</div>
+  </div>
+</div>
+
+Weight: 1.1
+
+Styleguide components.forms.form-groups
+*/
+
+.pt-form-group {
+  display: flex;
+  flex-direction: column;
+  margin: 0 0 $pt-grid-size;
+
+  label.pt-label {
+    margin-bottom: $pt-grid-size / 2;
+  }
+
+  .pt-form-helper-text {
+    margin-top: $pt-grid-size / 2;
+    color: $pt-text-color-muted;
+    font-size: $pt-font-size-small;
+  }
+
+  @each $intent, $color in $pt-intent-text-colors {
+    &.pt-intent-#{$intent} {
+      .pt-form-helper-text {
+        color: $color;
+      }
+    }
+  }
+
+  &.pt-inline {
+    flex-direction: row;
+
+    label.pt-label {
+      margin: 0 $pt-grid-size 0 0;
+      line-height: $pt-input-height;
+    }
+  }
+
+  &.pt-large label.pt-label {
```

Comment:
This rule is only accurate for the `pt-inline` case. If you're not inline, the line-height shouldn't change, otherwise we get inconsistent spacing between labels and inputs in the `input.pt-large` case and `.pt-form-group.pt-large` case